.env.example

```
@@ -7,14 +7,24 @@
 HOST_BIND_IP=127.0.0.1
 HOST_PORT=50080
+
+# ---------------------------------------------------------------------------
+# POSTGRESQL DATABASE SETTINGS
+# ---------------------------------------------------------------------------
+# These variables are used by the 'db' service in docker-compose.yml
+# and to construct the DATABASE_URL below.
+# For .env.test, it is recommended to change DB_NAME to 'django_db_test'.
+DB_NAME=django_db
+DB_USER=django_user
+DB_PASSWORD=django_password
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USER key

(UnorderedKey)